packages/react-storage/src/components/StorageBrowser/Views/Controls/Paginate.tsx

```diff
@@ -1,24 +1,64 @@
 import React from 'react';
 import { withBaseElementProps } from '@aws-amplify/ui-react-core/elements';
 
-import { StorageBrowserElements } from '../../context/elements';
+import {
+  ButtonElementProps,
+  IconElementProps,
+  PaginateVariant,
+  StorageBrowserElements,
+} from '../../context/elements';
 import { CLASS_BASE } from '../constants';
+import { PaginateStateContext, useControl } from '../../context/controls';
 
-const { Button, Icon, ListItem, Nav, OrderedList } = StorageBrowserElements;
+const {
+  Button: ButtonElement,
+  Icon: IconElement,
+  ListItem,
+  Nav,
+  OrderedList,
+  Text: TextElement,
+} = StorageBrowserElements;
 
 const BLOCK_NAME = `${CLASS_BASE}__paginate`;
+const PAGINATE_VARIANTS = [
```

Comment:
Is this going to be extended eventually/why is it in object form?

Reply:
May require extension in the future